Ship navigation guidance as skills, drop AGENTS.md/CLAUDE.md injection (#184)
alimanfoo/uncoded: .agents/skills/uncoded-code-navigation/SKILL.md

---
name: uncoded-code-navigation

> description: Use when finding, reading, or editing Python source in a codebase indexed by uncoded — locating a symbol, reading a definition, or checking references before a refactor, rename, or delete.

Author: ```suggestion
description: Use before searching, reading, or editing Python source in a codebase indexed by uncoded — locating a symbol, reading a definition, or checking references before a refactor, rename, or delete.
```

> description: 'Navigate Python source code in a codebase indexed by uncoded. Load .uncoded/namespace.yaml first, read .pyi stubs before source files, and apply the dispatch rule: symbol name to uncoded body/refs, pattern or phrase to grep.'

Author: The description should say when the skill should be applied, it's a trigger. Less what the skill mechanically does.

Author: Fixed in 38110bb. The description is now a trigger — "Use when finding, reading, or editing Python source in a codebase indexed by uncoded — locating a symbol, reading a definition, or checking references before a refactor, rename, or delete." The mechanical detail (load namespace.yaml, read stubs, dispatch rule) stays in the skill body where it belongs.

---

# Code Navigation

This codebase uses [uncoded](https://github.com/alimanfoo/uncoded) to maintain

> a symbol index over its source code, with two associated tools:

Author: ```suggestion
a symbol index over its source code, with two associated CLI tools:
```

`uncoded body` for reading a symbol's body and `uncoded refs` for finding
references.

> The point of this scaffolding is one rule.

Author: ```suggestion
```
